CMakeLists.txt

```diff
@@ -674,3 +674,21 @@ ExternalProject_Add(libbluray
 		-DCMAKE_PREFIX_PATH:PATH=${PREFIX}%3B%3B${PREFIX}/freetype%3B%3B${PREFIX}/libiconv%3B%3B${PREFIX}/libxml2
 )
 add_dependency_project_package(libbluray 1.0.1)
+
+ExternalProject_Add(flatbuffers
+	DOWNLOAD_DIR ${CMAKE_SOURCE_DIR}/downloads
+	URL http://mirrors.kodi.tv/build-deps/sources/flatbuffers-1.9.0.tar.gz
+	URL_HASH SHA256=5ca5491e4260cacae30f1a5786d109230db3f3a6e5a0eb45d0d0608293d247e3
+	CMAKE_ARGS
+		${ADDITIONAL_ARGS}
+		-DCMAKE_INSTALL_PREFIX:PATH=${INSTALL_PREFIX}
+		-DCMAKE_BUILD_TYPE=Release
```

Comment:
no, I want to be able choose the build type

Reply:
flatbuffers doesn't install the debug version of `flatc`. do we patch?

Reply:
https://github.com/google/flatbuffers/blob/v1.9.0/CMakeLists.txt#L295

Reply:
for a native package you can keep it

Reply:
yea, i figured for a native package release is fine